dmapiclient/base.py

```diff
@@ -76,6 +76,9 @@ def _delete_with_updated_by(self, url, data, user):
     def _request(self, method, url, data=None, params=None):
         if not self.enabled:
             return None
+        if self.base_url is None:
+            logger.info("{} has no URL configured".format(self.__class__.__name__))
```

Comment:
Hmm, so the reason that the constructor can't be the thing that fails, is because in practice the constructor is always called with no arguments (e.g. api:app/__init__.py:13), and the base URL is set later (via an init_app call).

Not sure why we have both ways of setting the configuration, but there it is.